influxql/engine.go

```diff
@@ -299,8 +299,13 @@ loop:
 	}
 
 	// Normalize rows and values.
+	// Convernt all times to timestamps
 	a := make(Rows, 0, len(rows))
 	for _, row := range rows {
+		for _, values := range row.Values {
+			t := time.Unix(0, values[0].(int64))
```

Comment:
I suppose it is possible if data got in somehow other than an int64 from UnixNano.  If it did, we are in big big trouble.  I guess worse case I could panic here.  The world would blow up and not sure what you would do about it.  Thoughts?